GCP/legos/gcp_remove_user_role/gcp_remove_user_role.py

```diff
@@ -0,0 +1,55 @@
+from typing import List
+from pydantic import BaseModel, Field
+from beartype import beartype
+import argparse
+from google.oauth2 import service_account
+
+class InputSchema(BaseModel):
+    role: str = Field(
+        title = "role",
+        description = "GCP user role to be removed"
+    )
+    member: str = Field(
+        title = "member",
+        description = "user's id to be removed"
+    )
+def modify_policy_remove_member(output):
+    if output is None:
+        return
+
+    print(output)
+
+@beartype
+def modify_policy_remove_member(policy, role: str, member: str):
+    """Removes a  member from a role binding.
+
+        :type role: string
+        :param role: user role to be removed.
+
+        :type member: string
+        :param member: user's id to be removed.
+
+        :rtype: confirmation of removal of role."""
+
+    service = discovery.build('iam', 'v1', credentials=credentials)
+    # The resource for which the policy is being requested.
+    resource = 'projects/my-project/serviceAccounts/my-service-account'
+    # TODO: Update placeholder value.
```

Comment:
Does this mean the resource path should be parameterized? meaning, should it be taken as an input parameter to this Lego? Also should it be part of the InputClass?

Reply:
Though the resource path is implicitly declared here, we can take it as a parameter from the user by including it in the input class. Shall I do that?

Reply:
Hi @SaxenaAnushka102  I wanted to check in to see if you had an ETA on creating the `resource` parameter so we can merge in your PR.

Thanks
Doug